content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,157 @@
+Title: Where to incorporate if you are a EU citizen
+Date: 2016-07-10
+Short_summary: We have lots of options as to where incorporate as EU citizens, let's look at them.
+Category: Business
+Authors: Vincent
+
+*Reading time: ~10 minutes.*
+
+We are very lucky as EU citizens to have access to 28 (well, soon 27 apparently) countries.
+To give a bit of context to this blog post, We Are Wizards is currently a company based in the UK, founded by 2 French and one German, all living in London at the moment.
+
+This article will try to be generalistic but we have a few requirements of our own that influenced this research. There might be inaccuracies in this article so don't take my word for it and contact companies/government in the countries before pulling the trigger.
+<!-- PELICAN_END_SUMMARY -->
+
+## Our requirements
+As mentioned in the introduction, we are French and German and the only common language is English so being able to do all (or at least the vast majority) government/bank interactions in English is a hard requirement.
+
+Since I personally like to travel and moving back to France/Germany is always a possibility, we also need to be able to incorporate in a country without living there. This is especially true if you are planning to be a digital nomad.
+
+Lastly, we want a business-friendly country with a good record on privacy and surveillance law as we run [Passopolis](https://passopolis.com/), an open-source password manager that relies on end-to-end encryption.
+
+To sum it up, our requirements are:
+
+- we need to be able to do things in English
+- we need to be able to incorporate without living there
+- we want its government to be privacy friendly and not authoritarian
+
+Your own requirements will obviously differ and some countries that we disregard in the next section might be the perfect fit for you. Or you might want to get out of the EU to not deal with [VAT MOSS](https://blog.wearewizards.io/looking-at-payments-solution-for-saas-in-europe) like [Ghost](https://blog.ghost.org/moving-to-singapore/) for example.
+
+## Countries selection
+
+France, Germany, Spain, Portugal, the Netherlands and Denmark fail as I couldn't find all the needed information on incorporation in English and the local language is required for all interactions.
```

Comment:
I'm surprised, Berlin is known for its startup friendly environment, I'd have thought that German wasn't a requirement to open a company there.